server/sonar-server/src/main/java/org/sonar/server/notification/ws/AddAction.java

```diff
@@ -45,26 +45,29 @@
 import static org.sonar.core.util.stream.MoreCollectors.toList;
 import static org.sonar.server.notification.NotificationDispatcherMetadata.GLOBAL_NOTIFICATION;
 import static org.sonar.server.notification.NotificationDispatcherMetadata.PER_PROJECT_NOTIFICATION;
-import static org.sonar.server.ws.WsUtils.checkFound;
-import static org.sonar.server.ws.WsUtils.checkRequest;
 import static org.sonar.server.notification.ws.NotificationsWsParameters.ACTION_ADD;
 import static org.sonar.server.notification.ws.NotificationsWsParameters.PARAM_CHANNEL;
 import static org.sonar.server.notification.ws.NotificationsWsParameters.PARAM_LOGIN;
 import static org.sonar.server.notification.ws.NotificationsWsParameters.PARAM_PROJECT;
 import static org.sonar.server.notification.ws.NotificationsWsParameters.PARAM_TYPE;
+import static org.sonar.server.ws.WsUtils.checkFound;
+import static org.sonar.server.ws.WsUtils.checkRequest;
 
 public class AddAction implements NotificationsWsAction {
   private final NotificationCenter notificationCenter;
   private final NotificationUpdater notificationUpdater;
+  private final Dispatchers dispatchers;
   private final DbClient dbClient;
   private final ComponentFinder componentFinder;
   private final UserSession userSession;
   private final List<String> globalDispatchers;
   private final List<String> projectDispatchers;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "projectDispatchers" private field. ([squid:S1068](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fjl%2FSONAR-8964_global_notif_on_sonarcloud&issues=AWFxXZ532VQm03fG7H_k&open=AWFxXZ532VQm03fG7H_k)

[ISSUE_KEY]: AWFxXZ532VQm03fG7H_k